schema/spacewalk/common/tables/endpoint.sql

```diff
@@ -0,0 +1,26 @@
+--
+-- Copyright (c) 2025 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE TABLE access.endpoint (
+    id              BIGINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    class_method    VARCHAR NOT NULL,
+    endpoint        VARCHAR NOT NULL,
+    http_method     VARCHAR NOT NULL,
+    scope           CHAR(1) NOT NULL
+                        CHECK (scope in ('A', 'W')),
+    authorized      BOOLEAN NOT NULL DEFAULT true,
```

Comment:
I would make it with a more explicit name, like auth_required, as proposed in the RFC.